packages/api-client/index.ts

```diff
@@ -94,6 +94,23 @@ interface SearchClass {
   coreqs?: INEUAndReq | INEUOrReq;
   maxCredits: number;
   minCredits: number;
+  termId: string;
+}
+
+function occurrencesToCourseByCatalogYear(
+  occurrences: SearchClass[],
```

Comment:
nit: occurrences, the way it is used, could be undefined or of length 0.  have a check for undef and length > 0, throw error otherwise